doc/source/release-notes/Percona-Server-5.6.33-79.0.rst

```diff
@@ -0,0 +1,118 @@
+ the ``CREATE TABLE mysql.server`` statement would attempt to create an
+ |InnoDB| table and fail because ``mysql_system_tables.sql`` script omitted
+ explicit engine setting for this table. Bug fixed :bug:`1600056`.
+ 
+ :ref:`audit_log_plugin` malformed record could be written after
+ :variable:`audit_log_flush` was set to ``ON`` in ``ASYNC`` and ``PERFORMANCE``
+ modes. Bug fixed :bug:`1613650`.
+
+ A race condition between :ref:`handlersocket_page` and server shutdown could
+ lead to a server stall if shutdown was issued immediately after
+ :ref:`handlersocket_page` startup. Bug fixed :bug:`1617198`.
+
+ :ref:`handlersocket_page` could access freed memory on startup. Bug fixed
+ :bug:`1617998`.
+
+ Running ``LOCK TABLES`` with :ref:`threadpool` could lead to a deadlock. Bug
+ fixed :bug:`1619559`.
+
+ Fixed memory leaks in :ref:`audit_log_plugin`. Bug fixed :bug:`1620152`
+ (upstream :mysqlbug:`71759`).
+
+ Server could crash due to a ``glibc`` bug in handling short-lived detached
+ threads. Bug fixed :bug:`1621012` (upstream :mysqlbug:`82886`).
+
+ ``QUERY_RESPONSE_TIME_READ`` and ``QUERY_RESPONSE_TIME_WRITE`` were returning
+ common table data if accessed  through a name that is not full uppercase. Bug
+ fixed :bug:`1552428`.
+
+ Fixed memory leaks in :ref:`handlersocket_page`. Bug fixed :bug:`1617949`.
+
+ ``KILL QUERY`` was not behaving consistently and it would hang in some cases.
+ Bug fixed :bug:`1621046` (upstream :mysqlbug:`45679`).
+
+ Cipher ``ECDHE-RSA-AES128-GCM-SHA256`` was listed in the `list
+ <https://dev.mysql.com/doc/refman/5.6/en/secure-connection-protocols-ciphers.html>`_
+ of supported ciphers but it wasn't supported. Bug fixed :bug:`1622034`
+ (upstream :mysqlbug:`82935`). 
+
+ Successful doublewrite recovery was showed as a warning in the error log. Bug
+ fixed :bug:`1622985`.
+
+ Variable :variable:`query_cache_type` couldn't be set to ``0`` if it was
+ already set to that value. Bug fixed :bug:`1625501` (upstream
+ :mysqlbug:`69396`).
+
+ LRU manager thread could run too long on a server shutdown. Bug fixed
+ :bug:`1626069`.
+
+ ``tokudb_default`` was not recognized by |Percona Server| as valid row format.
+ Bug fixed :bug:`1626206`.
+
+ |InnoDB| ``ANALYZE TABLE`` didn't remove its table from the background
+ statistics processing queue. Bug fixed :bug:`1626441` (upstream
+ :mysqlbug:`71761`).
+
+ Upstream merge for :mysqlbug:`81657` to 5.6 was incorrect. Bug fixed
+ :bug:`1626936` (upstream :mysqlbug:`83124`). 
+
+ Fixed multi-threaded slave memory leaks that happened in case of thread create
+ failure. Bug fixed :bug:`1619622` (upstream :mysqlbug:`82980`).
+
+ Threads with ``GLOBAL READ`` lock (executing ``FLUSH TABLES WITH READ LOCK``)
+ are now put into high priority queue. Bug fixed :bug:`1374930`.
+
+ Shutdown waiting for a purge to complete was undiagnosed for the first minute.
+ Bug fixed :bug:`1616785`.
+
+ |InnoDB| buffer merge has been optimized when it's reading a page from a disk.
```

Comment:
Unnecessary |InnoDB| change buffer merge attempts are now skipped upon reading disk pages of non-applying types